runbotics/runbotics-desktop/src/action/sharepoint/excel/types.ts

```diff
@@ -36,10 +36,11 @@ export type SharepointGetExcelCellActionInput = {
 };
 export type SharepointExcelGetCellActionOutput = any;
 
-export type SharepointExcelGetRangeActionInput = {
-    range: string;
+export type SharepointExcelGetCellsActionInput = {
+    startCell: string;
+    endCell: string;
 };
-export type SharepointExcelGetRangeActionOutput = any;
+export type SharepointExcelGetCellsActionOutput = any;
```

Comment:
try to specify the type if possible, otherwise it's ok

Reply:
i think we have ExcelCell or sth like that